File: problema 13/fortran/problema_13.f90
program contarCeros
    implicit none
    integer :: arreglo(5,5)
    integer :: i, j, contador
    real :: numAleatorio

    contador = 0
    call random_seed

    ! Llenar el arreglo y contar los ceros
    do i = 1, 5
        do j = 1, 5
            call random_number(numAleatorio)
            arreglo(i,j) = int(numAleatorio * 10)
            if (arreglo(i,j) == 0) then
                contador = contador + 1
            endif
        end do
    end do

    ! Mostrar el arreglo (opcional)
    do i = 1, 5
        do j = 1, 5
            write(*, '(I2)', advance="no") arreglo(i,j)
        end do
        write(*, *)
    end do

    ! Mostrar el número de ceros
    write(*, *) 'Número de ceros en el arreglo: ', contador

end program contarCeros

File: problema 13/c-languaje/problema_13.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main() {
    int arreglo[5][5];
    int contador = 0;
    int i, j;

    // Inicializar la semilla para números aleatorios
    srand(time(NULL));

    // Llenar el arreglo y contar los ceros
    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            arreglo[i][j] = rand() % 10;  // Números entre 0 y 9
            if (arreglo[i][j] == 0) {
                contador++;
            }
        }
    }

    // Mostrar el arreglo (opcional)
    for (i = 0; i < 5; i++) {
        for (j = 0; j < 5; j++) {
            printf("%d ", arreglo[i][j]);
        }
        printf("\n");
    }

    // Mostrar el número de ceros
    printf("Número de ceros en el arreglo: %d\n", contador);

    return 0;
}